python/sdk/tests/test_build.py

```diff
@@ -108,14 +108,21 @@ def test_download(self, unwrap):
 
     @pytest.mark.order(-2)
     def test_delete_test(self, unwrap):
-        unwrap(self.build.delete_test)(self.build, test_id=pytest.test_id)
+        unwrap(self.build.delete_test)(self.build, test_id=pytest.test_id, purge=False)
         res = unwrap(self.detect.get_test)(self.detect, test_id=pytest.test_id)
         pytest.expected_test['tombstoned'] = res['tombstoned']
 
         diffs = check_dict_items(pytest.expected_test, res)
         assert not diffs, json.dumps(diffs, indent=2)
         assert parse(res['tombstoned']) <= datetime.utcnow() + timedelta(minutes=1)
 
+    def test_purge_test(self, unwrap):
```

Comment:
this is weird to throw in one random semi-independent test amongst everything. prefer you just tack it on to the rest of the test suite (purge the just-tombstoned test above), but if you're keen for independent tests, start some new files that we can slowly start porting to

Reply:
👍 I was concerned deleting it would break other tests that depend on the test existing

Reply:
the `@pytest.mark.order(-2)` on `test_delete_test` means "run it second to last" (purge_account is last). so you can adjust that one and put the order decorator on this one so that purge_test runs second to last and delete_test runs third to last

Reply:
ah ok that should work then